### PR TITLE
Improved test cases and seed data fixed

### PR DESCRIPTION
**Pull request by DeepikaSinghD**
This PR does not contain any new test cases for the story CVSB-18783 https://jira.dvsacloud.uk/browse/CVSB-18783
There are some test cases which are improved and the test data is fixed.

### CODE REVIEW

**Review by bhavia-dvsa — APPROVED** [review]
Approving this PR

File: src/main/resources/loader/develop/test-results_roadworthiness_hgv_pass_7675.json [hunk 1]
```diff
@@ -6,8 +6,8 @@
   "testerName":"Deepika Singh",
   "testerStaffId":"d0b09235-7234-4bed-b628-e4c2a1fcb414",
   "testerEmailAddress":"deepikasingh@dvsagov.onmicrosoft.com",
-  "testStartTimestamp":"2019-12-12T10:41:53.012Z",
-  "testEndTimestamp":"2019-12-12T10:42:58.970Z",
+  "testStartTimestamp":"2020-12-12T10:41:53.012Z",
```
Comment (IuliaDVSA) [inline src/main/resources/loader/develop/test-results_roadworthiness_hgv_pass_7675.json:9]: Why were the dates updated with 2020 not 2021?

File: src/test/java/testresults/PostTestResultsPozDefectsLvlCancelled.java [hunk 2]
```diff
@@ -36,29 +39,91 @@ private void validateSavedDataOld() {
...
+        JsonPathAlteration alterationSystemNumber = new JsonPathAlteration("$.systemNumber", randomSystemNumber,"","REPLACE");
+        JsonPathAlteration alterationTestResultId = new JsonPathAlteration("$.testResultId", randomTestResultId, "", "REPLACE");
+        JsonPathAlteration alterationImDescription = new JsonPathAlteration("$.testTypes[0].defects[0].imNumber", RandomStringUtils.randomNumeric(5), "", "REPLACE");
+        JsonPathAlteration alterationTestCode = new JsonPathAlteration("$.testTypes[0].testCode", "","","DELETE");
+        JsonPathAlteration alterationTestNumber = new JsonPathAlteration("$.testTypes[0].testNumber", "","","DELETE");
+        JsonPathAlteration alterationLastUpdatedAt = new JsonPathAlteration("$.testTypes[0].lastUpdatedAt", "","","DELETE");
+        JsonPathAlteration alterationCreatedAt = new JsonPathAlteration("$.testTypes[0].createdAt", "","","DELETE");
+        JsonPathAlteration alterationCertificateLink = new JsonPathAlteration("$.testTypes[0].certificateLink", "","","DELETE");
+        JsonPathAlteration alterationVehicleId = new JsonPathAlteration("$.vehicleId", "","","DELETE");
+
```
Comment (IuliaDVSA) [inline src/test/java/testresults/PostTestResultsPozDefectsLvlCancelled.java:61]: Please remove the additional blank line.

File: src/test/java/testresults/PostTestResultsPozDefectsLvlCancelled.java [hunk 3]
```diff
@@ -36,29 +39,91 @@ private void validateSavedDataOld() {
...
+        JsonPathAlteration alterationSystemNumber = new JsonPathAlteration("$.systemNumber", randomSystemNumber,"","REPLACE");
+        JsonPathAlteration alterationTestResultId = new JsonPathAlteration("$.testResultId", randomTestResultId, "", "REPLACE");
+        JsonPathAlteration alterationImDescription = new JsonPathAlteration("$.testTypes[0].defects[0].imDescription", RandomStringUtils.randomAlphanumeric(14), "", "REPLACE");
+        JsonPathAlteration alterationTestCode = new JsonPathAlteration("$.testTypes[0].testCode", "","","DELETE");
+        JsonPathAlteration alterationTestNumber = new JsonPathAlteration("$.testTypes[0].testNumber", "","","DELETE");
+        JsonPathAlteration alterationLastUpdatedAt = new JsonPathAlteration("$.testTypes[0].lastUpdatedAt", "","","DELETE");
+        JsonPathAlteration alterationCreatedAt = new JsonPathAlteration("$.testTypes[0].createdAt", "","","DELETE");
+        JsonPathAlteration alterationCertificateLink = new JsonPathAlteration("$.testTypes[0].certificateLink", "","","DELETE");
+        JsonPathAlteration alterationVehicleId = new JsonPathAlteration("$.vehicleId", "","","DELETE");
+
```
Comment (IuliaDVSA) [inline src/test/java/testresults/PostTestResultsPozDefectsLvlCancelled.java:107]: Please remove the additional blank line.

**Review by IuliaDVSA — CHANGES_REQUESTED** [review]
Please see comments on  src/test/java/testresults/PostTestResultsPozDefectsLvlCancelled.java

src/main/resources/loader/develop/test-results_roadworthiness_hgv_pass_7675.json

File: src/main/resources/loader/develop/test-results_roadworthiness_hgv_pass_7675.json (hunk 1 above)
Comment (DeepikaSinghD) [reply src/main/resources/loader/develop/test-results_roadworthiness_hgv_pass_7675.json:9]: The tests date were outdated hence updated.

File: src/test/java/testresults/PostTestResultsPozDefectsLvlCancelled.java (hunk 2 above)
Comment (DeepikaSinghD) [reply src/test/java/testresults/PostTestResultsPozDefectsLvlCancelled.java:61]: This is not a problem, it is used to simply differentiate between code.

File: src/test/java/testresults/PostTestResultsPozDefectsLvlCancelled.java (hunk 3 above)
Comment (DeepikaSinghD) [reply src/test/java/testresults/PostTestResultsPozDefectsLvlCancelled.java:107]: This is not a problem, it is used to simply differentiate between code.

**Review by IuliaDVSA — APPROVED** [review]

**Pull request merged**